demisto_sdk/commands/common/hook_validations/layout.py

```diff
@@ -237,3 +250,6 @@ def is_incident_field_exist(self, id_set_file, is_circle) -> bool:
             if self.handle_error(error_message, error_code, file_path=self.file_path):
                 return False
         return True
+
+    def is_id_equals_name(self) -> bool:
+        return True
```

Comment:
add a comment explaining the hard-coded value (there is no name in legacy layouts)

Reply:
i dont quiet understated what you mean